### ISSUE TITLE
Some Questions about MMLU Dataset

### ISSUE THREAD

**Issue by JamesSand** [shizhediao/R-Tuning#4]
Hi, 
Thank you for your great work! 
In your paper, you have conducted experiment on MMLU dataset, and it seems that you have done R-tunning on MMLU dataset. 
I found a MMLU dataset on huggingface, which link is https://huggingface.co/datasets/lukaemon/mmlu and I saw that there are only 4 training samples for each category. Just want to confirm that did you use just 4 samples for each categories for fine-tuning? Or you have re-split this dataset and using more samples for finetuning? I am concerning 4 samples might be too few...

Thank you for your time and patience. Looking forward to your reply.

**Comment by shizhediao** [comment]
Hi,
Thanks for your attention!
Yes, we re-split the dataset to include more training data. 
For MMLU dataset, since there are 57 categories of tasks, we use the first half (28 categories) for in-domain data and the last half (29 categories) for out-of-domain data. For the in-domain data, we divide the test set of each category into half of equal size for training and testing.

Hope this helps. Thanks!

**Comment by JamesSand** [comment]
Thank you for your nice response!
